Formula/fontforge.rb

```diff
@@ -105,6 +106,9 @@ def caveats; <<-EOS.undent
 
     Alternatively, install with Homebrew-Cask:
       brew cask install fontforge
+
+    As a last resort, you may try the *unsupported* option:
+      brew install fontforge --with-x11
```

Comment:
If you tag me in something it'll likely get a reply, I'm just not spending much time actively looking for stuff to do.

I have no objection to the X11 dep going in as optional personally, at least until https://github.com/fontforge/fontforge/pull/2782 is resolved positively (fingers crossed) and that's available in a release. There's a decent amount of optional X11 stuff in core.

It'd be a fair claim that most people doing `brew install fontforge` want at least a GUI optionally available. The alternative if upstream is planning to keep making `.app`s is to remove this entirely in favour of the Cask.